examples/sklearnex/spmd_pca.py

```diff
@@ -0,0 +1,42 @@
+# ===============================================================================
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ===============================================================================
+
+import numpy as np
+from mpi4py import MPI
+import dpctl
+from sklearnex.spmd.decomposition import PCA as PCASpmd
+
+
+def get_data(data_seed, params_spmd):
+    drng = np.random.default_rng(data_seed)
+    X = drng.uniform(-7, 7, size=(params_spmd["ns"], params_spmd["nf"]))
+    y = drng.uniform(-7, 7, size=(params_spmd["ns"], params_spmd["nr"]))
```

Comment:
PCA is a dimension reduction algorithm, thus it needs only `X` value.

Reply:
https://github.com/intel/scikit-learn-intelex/blob/1674bb9719cc8cb42adce2dc8ef48ffe5f68b608/onedal/decomposition/pca.py#L46 

x, y both are kept for purpose of alignment. Should I make changes to remove y?